scripts/sample-reads-randomly.py

```diff
@@ -108,6 +108,10 @@ def main():
         output_filename = output_file.name
     else:
         filename = args.filenames[0]
+        if filename == '/dev/stdin' or filename == '-':
```

Comment:
` ('-' in args.input_filenames or '/dev/stdin' in args.input_filenames)` would be more consistent with other changes you've made